.github/workflows/automation-ondemand-tests.yml

```diff
@@ -111,17 +115,26 @@ jobs:
       matrix:
         tests:
           - name: chaos
+            suite: chaos
             nodes: 5
             os: ubuntu-latest
             pyroscope_env: ci-automation-on-demand-chaos
             network: SIMULATED
             command: -run ^TestAutomationChaos$ ./chaos
           - name: reorg
+            suite: reorg
             nodes: 1
             os: ubuntu-latest
             pyroscope_env: ci-automation-on-demand-reorg
             network: SIMULATED_NONDEV
             command: -run ^TestAutomationReorg$ ./reorg
+          - name: upgrade
+            suite: smoke
+            nodes: 1
+            os: ubuntu-latest
+            pyroscope_env: ci-automation-on-demand-upgrade
```

Comment:
Is this the same pyroscope service that lets you view memory usage of apps? If so can we use this to debug the slow memory leak?